openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx

```diff
@@ -31,6 +31,7 @@ const DeleteWidgetModal = ({
   allowSoftDelete = true,
   visible,
   deleteMessage,
+  softDeleteMessage = '',
```

Comment:
It's a postFix for an existing message name it accordingly else it will be confused with it will replace current message completely